### PR TITLE
Remove Global Table attributes when creating or updating records, als…

### PR DESCRIPTION
**Pull request by msluther**
…o lint cleanup and creating changelog.

Per <https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/globaltables_HowItWorks.html>:

> **Important**
> To keep your table data in sync, global tables automatically create the following attributes for every item:
> 
> `aws:rep:deleting`
> 
> `aws:rep:updatetime`
> 
> `aws:rep:updateregion`
> 
> Do not modify these attributes or create attributes with the same name.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -142,6 +143,20 @@ class Dynastar {
...
   }
+
+  /**
+   * Removes attributes that are created by Global Tables and should never be written.
+   * @see https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/globaltables_HowItWorks.html
+   * @param {Object} data The model data
+   * @returns {Object} The model data without the global-table attributes
+   */
+  _omitGlobalTableData(data) {
+    return omit(data, [
```
Comment (Swaagie) [inline index.js]: nitpick `delete data['aws:rep:deleting']`? No need to change, just wondering what `omit` provides?

Comment (msluther) [reply index.js]: oh good point. let me look. i just copied over stuff from bffs.  I believe this doesn't affect the initial object like `delete` would, but I want to double check that.

Comment (msluther) [reply index.js]: updated the new tests to show the initial spec is unchanged.

Comment (msluther) [reply index.js]: Though I supposed I could have done the same still by using delete over the newly created object. :|

Comment (msluther) [reply index.js]: There I fixed it. lodash.omit is no more. Thanks :)

**Pull request merged**